OffbeatStore/utility_functions/notification.py

```diff
@@ -0,0 +1,22 @@
+from flask import g
```

Comment:
It's better if we don't use the flask app here. 
If you need the user pass it to the method or do the check before.